Gopkg.toml

```diff
@@ -9,7 +9,7 @@ required = [
 ]
 
 
-[[constraint]]
+[[override]]
```

Comment:
This is to keep the CSI spec at v1.0.0 since we have v1.1.0 at csi-lib-utils

Reply:
Do you still need this override now that version is 1.1.0?

Reply:
This should be a constraint, not override because it is in sync with csi-lib-utils.

Reply:
fixed